src/Utils/helper.ts

```diff
@@ -0,0 +1,14 @@
+require("dotenv").config({ path: `${__dirname}/.env` });
+const pg = require("pg");
+
+const {
+  APP_ENV,
+  DB_HOST,
+  DB_USER,
+  DB_PASS
+} = process.env;
+
+if (process.env.NODE_ENV === "development")
+console.log(process.env.APP_ENV)
```

Comment:
we shouldn't have `console.log()` in production code

Reply:
i forgot to exclude this when pushing,i was testing if it worked